src/test/java/GuavaProgramming/PredicateUtilityTest.java

```diff
@@ -0,0 +1,35 @@
+package GuavaProgramming;
+
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
```

Comment:
dont use general imports (*), use specific imports. 
If you are using intellij please consider following settings.
<img width="1035" alt="intellij autoimport" src="https://cloud.githubusercontent.com/assets/8583715/18410298/67f750d8-775e-11e6-9f62-7d48edb00e48.png">